back-end/API/tests/test_employee_jobOffer.py

```
@@ -0,0 +1,470 @@
+            "address": "123 rue de la rue",
+            "isTemporary": False,
+            "cityId": 1,
+        })
+es =EmploymentSchedule(id=1, description="Temps plein")
+sp1 = StudyProgram(id=1, name="Informatique")
+sp2 = StudyProgram(id=2, name="Génie logiciel")
+
+
+def VerifyData(jobOfferData):
+    for data in job_offer1_data:
+        if (data not in jobOfferData) and (type(jobOfferData[data]) != type(job_offer1_data[data])) :
```

Ruff (0.15.13)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)